apps/meeting/src/components/Spinner/index.tsx

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
The Copyright here should be 2021.
In another repo we drop the year in Copyright, not sure if we need to do that in this repo as well. But for now we should keep the new file as 2021,

Reply:
Updated